app/functions/functions.php

```diff
@@ -130,3 +132,8 @@ function submit_lost(){
 function print_tag($var, $default = '&mdash;') {
 	echo isset($var) ? $var : $default;
 }
+
+function parseUrl() {
+// Trim URL, Sanitize and explode to array at each /
+	return $url = explode('/pet-details/', filter_var(rtrim($_SERVER['REQUEST_URI'], '/'), FILTER_SANITIZE_URL));
+}
```

Comment:
Actually everything here looks like 6 spaces. Smells of SO copy and paste to me ;P